### PR TITLE
[LAYOUTS] Layout-Container

### PR DESCRIPTION
**Pull request by mikorosales**
## Issue Link
[LMS-304](https://framgiaph.backlog.com/view/LMS-304)

## Definition of Done
container.tsx should be created under demo > layouts >

## Steps to reproduce
- http://localhost:3000/demo/layouts/container

## Affected Components / Functionalities / Page
n/a

## Test Cases
_will update soon..._

## Notes
n/a

## Screenshots
image
image

### CODE REVIEW

**Comment by Marielleregner7** [comment]
Working:
[] Child elements should be wrapped inside a layout container

Comment:
[] Unnecessary horizontal Lines are visible to both pages.
image

image

File: client/styles/globals.css [hunk 1]
```diff
@@ -1,112 +1,3 @@
...
-  color: inherit;
-  text-decoration: none;
-}
-
-@media (prefers-color-scheme: dark) {
-  html {
-    color-scheme: dark;
-  }
-}
+@tailwind utilities;
```
Comment (escbooster12-dev) [inline client/styles/globals.css:3]: - add newline

File: client/src/shared/components/layouts/LayoutContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+interface LayoutContainerProps {
+    children: ReactNode
+}
+
+const LayoutContainer = ({ children }: LayoutContainerProps) => {
+    return (
+        <>
+            <div className="container mx-auto px-20">
+                <div className='border border-black'>
```
Comment (escbooster12-dev) [inline client/src/shared/components/layouts/LayoutContainer.tsx:11]: - no need to add border. just follow background color in the figma

File: client/src/pages/OtherPage.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const OtherPage = () => {
```
Comment (escbooster12-dev) [inline client/src/pages/OtherPage.tsx:3]: - where this file being used?

**Review by escbooster12-dev — COMMENTED** [review]
Comments: 3
Iteration: 1

**Review by Marielleregner7 — APPROVED** [review]
**Approved**

- Able to create a new page
- The layout container can be seen in the page created
-  Unnecessary lines that I have seen in the first PR are no longer visible

Comments: 1
Test time: 45 mins
Iterations: 1

**Review by escbooster12-dev — APPROVED** [review]

**Review by Marielleregner7 — APPROVED** [review]
APPROVED
- [x] Container is visible with the same color in the wireframe and can be seen in http://localhost:3000/demo/layouts/container
- [x] container.tsx should be created under demo > layouts >

Comments: 2
Test time: 30 mins
Iterations: 2

**Review by escbooster12-dev — APPROVED** [review]
Comments: 7
iteration: 3

**Pull request merged**